tools/build/scripts/copy-assets.js

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env node
+
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+"use strict"
+
+const argv = require("yargs").argv;
+const fs = require("fs-extra");
+const path = require("path");
+
+// example usage
+// betools copy-assets --nodeModulesDir=../../../../
+
+const packageJsonDir = (argv.packageJsonDir === undefined) ? "." : argv.packageJsonDir;
+const nodeModulesDir = (argv.nodeModulesDir === undefined) ? "." : argv.nodeModulesDir;
```

Comment:
`const nodeModulesDir = argv.nodeModulesDir ?? ".";` would be more readable. It's supported by Node.js 14.0 and above, I don't think our backends should use anything older than that.

Reply:
Nice, yeah I specifically didn't because I saw Arun's PR here: https://github.com/imodeljs/create-react-app/pull/72 but if it's safe to support I'll make that fix now!

Reply:
Ok updated. I also added the { rercursive: true } option to the mkdirSync in case the lib folder doesn't exist yet

Reply:
alright, one more thing I just noticed: this file uses tabs, please change it to spaces for consistency

Reply:
done